### ISSUE TITLE
device-side assert triggered at /opt/conda/conda-bld/pytorch_1535493744281/work/aten/src/THC/generic/THCStorage.cpp:36

### ISSUE THREAD

**Issue by chun-chun123** [STVIR/pysot#263]
I use the training code supplied without changing.But it shows this error.
I have found blogs for this error and known it mainly because the label computing loss may have  negative value.
But I download the VID dataset in the official web and use the applied code to change it  to JSON.
Why there is still this error ?

**Comment by smile-fyl** [comment]
1

**Comment by chun-chun123** [comment]
I have solved this promblem.
I found that when others run their codes that occupy plenty of memory,it will show this error.
